.github/workflows/test.yml

```diff
@@ -33,10 +33,10 @@ jobs:
         run: pip install .
       - name: Install Dependencies
         run: pip install -r requirements/test.txt
-      - name: Run pytest
-        run: pytest -v --cov=codemodder
-      - name: Report coverage
-        run: coverage report -m --fail-under=95
+      - name: Run unit tests
+        run: pytest -v --cov-fail-under=95 --cov=codemodder tests --numprocesses auto
+      - name: Run integration tests
```

Comment:
Eventually I think this could just happen in a different action but for now this looks good.